server.js

```diff
@@ -1,29 +1,135 @@
 import express from "express";
 import cors from "cors";
+import netflixData from "./data/netflix-titles.json";
+console.log(netflixData.length);
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+// Defines the port the app will run on.
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
+// Import a module that helps list all registered endpoints in an Express app.
+const listEndPoints = require("express-list-endpoints");
+
+// Enable Cross-Origin Resource Sharing (CORS) for all routes.
 app.use(cors());
-app.use(express.json());
+app.use(express.json()); // Parse incoming JSON requests.
 
-// Start defining your routes here
+// Start defining your routes here.
+// Home route that returns a list of all defined endpoints.
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(listEndPoints(app));
+});
```

Comment:
To help the user and to document more as you mentioned maybe something like this might be helpful to add to the "homepage"? 
const documentation = {
    endpoints: listEndpoints(app),
    additionalInfo: {
       "/browse/:browse": {
        description: "To get a list of xxxxxxxx"
        },
        example: "xxx example-of-url here",
      },
    },
  };

  res.json(documentation);